src/System.Private.Xml/tests/Xslt/XslTransformApi/CXslTransform.cs

```diff
@@ -523,13 +506,18 @@ public void XmlResolver7(InputType inputType, ReaderType readerType, TransformTy
             {
                 _output.WriteLine(e.ToString());
                 _output.WriteLine("Could not copy file to local. Some other issues prevented this test from running");
-                return; //return TEST_SKIPPED;
+                throw;
```

Comment:
Agreed: I would like to remove those too, this is just a matter of the scope of the change for this PR.